data/secscan_model/secscan_v4_model.py

```diff
@@ -178,16 +170,14 @@ def eligible_manifests(base_query):
         # TODO(alecmerdler): Filter out any `Manifests` that are still being uploaded
         def not_indexed_query():
             return (
-                eligible_manifests(Manifest.select())
-                .switch(Manifest)
+                Manifest.select()
                 .join(ManifestSecurityStatus, JOIN.LEFT_OUTER)
                 .where(ManifestSecurityStatus.id >> None)
```

Comment:
`ManifestSecurityStatus` objects can be created without an ID?

Reply:
This is a left outer join; if the MSS row doesn't exist, "it"'s ID will be NULL

Reply:
Ahhh, that makes sense. Thanks for pointing that out.